samcli/local/init/templates/cookiecutter-aws-sam-hello-dotnet/{{cookiecutter.project_name}}/README.md

```diff
@@ -7,7 +7,21 @@ This is a sample template for {{ cookiecutter.project_name }}
 * AWS CLI already configured with at least PowerUser permission
 * [Docker installed](https://www.docker.com/community-edition)
 * [SAM Local installed](https://github.com/awslabs/aws-sam-cli)
-* [DotNet Core installed](https://www.microsoft.com/net/download)
+* [.NET Core installed](https://www.microsoft.com/net/download)
+
+The sample template for {{ cookiecutter.project_name }} uses Cake. The Cake dependencies are listed below, per Operating System.
```

Comment:
Is Cake really necessary? Can't we just use dotnet CLI's native commands instead?

Reply:
Hi @sanathkr ,

To answer your question, no is not absolutely necessary. However, from a dev perspective, it is a useful abstraction since gives you the ability to use the same commands to build your solution as the CI server.

Using Cake in .NET is similar to use Maven in Java (although Maven as the package manager component, where Cake uses NuGet, Paket, ..., for .NET dependencies). However, comparing to SAM Java version, it uses Maven to run the tests, where it also could use `java` CLI.

Does it make sense?

Reply:
Agree, this is quite useful as its also cross platform.

Reply:
According to [this thread](https://github.com/cake-build/cake/issues/1751), Cake can run on .NET Core without Mono. They also link [this script](https://github.com/devlead/BitbucketPipelinesShield/blob/master/build.sh) for bootstrapping Cake and getting the Cake.CoreCLR as opposed to relying on the broader .NET Framework (which Mono provides). 

They've also release it as a .NET CLI tool, as of v0.30.0 (blog post [here](https://cakebuild.net/blog/2018/08/cake-v0.30.0-released)). I'm thinking with this new info and the improvements to Cake, we can remove the reliance on Mono and clean up this experience. 

Thoughts?

Reply:
@mikemorain you are correct, Cake no longer requires Mono, and we should probably remove that section. I'd still like to put something here about Mono as many tools we use for cross platform development and IDE support (VS Code and Rider for example) rely on it for debugging purposes.